ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -95,8 +98,16 @@ const PrepareBridgePage = () => {
       destChainId: toChain?.chainId
         ? Number(hexToDecimal(toChain.chainId))
         : undefined,
+      insufficientBal: Boolean(providerConfig?.rpcUrl?.includes('tenderly')),
```

Comment:
What does this do? If the rpc is Tenderly they always have insufficient balance?

Reply:
That's right. This needs to be sent to the backend so it adds the necessary simulation overrides. Otherwise quotes are not returned when the wallet's real balance is less than the tenderly balance

Reply:
Can we just have a comment about that for future readers of the code?